data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java

```diff
@@ -72,7 +77,9 @@ public PhysicalParameter getPhysicalParameter() {
         if (direction == Direction.DIRECT) {
             return Optional.empty();
         } else { // Direction.OPPOSITE and Direction.BOTH
-            return Optional.of(-convert(getAbsoluteMax(), unit, requestedUnit));
+            return Optional.of(
+                    convert(frmInMW, Unit.MEGAWATT, requestedUnit)
+                    - convert(getAbsoluteMax(), unit, requestedUnit));
```

Comment:
add some unit tests to check that the returned value is ok in different cases ? 
- FRM = 0
- FRM != 0
- different units

Reply:
this is partially done, because testing a threshold conversion cannot be done in a unit test (it recquires a synchronisation with a network)